src/ui/Search/searchinfilesworker.cpp

```diff
@@ -22,53 +22,43 @@ SearchInFilesWorker::~SearchInFilesWorker()
 
 void SearchInFilesWorker::run()
 {
-    // Search string converted to a regex
-    QString rawSearch = frmSearchReplace::rawSearchString(m_string, m_searchMode, m_searchOptions);
+    FileSearchResult::SearchResult searchResult;
+    searchResult.search = m_string;
 
     QFlags<QRegularExpression::PatternOption> options = QRegularExpression::NoPatternOption;
```

Comment:
The multiline regex option was actually a good thing. Change line 28 to this:
        
    QFlags<QRegularExpression::PatternOption> options = QRegularExpression::MultilineOption;